BP/localization/english/culture/BP_cultures_l_english.yml

```diff
@@ -0,0 +1,198 @@
+﻿l_english: 
+ # Baltic Group
+ vistula_veneti:0 "Vistula-Veneti"
+ vistula_veneti_collective_noun:0 "Vistula-Veneti"
+ przeworsk:0 "Przeworsk"
+ przeworsk_collective_noun:0 ""
+ galindian:0 "Galindian"
+ galindian_collective_noun:0 "Galindians"
+ aesti:0 "Aesti"
+ aesti_collective_noun:0 "Aesti"
+ # Brythonic group
+ briton:0 "Briton"
+ briton_collective_noun:0 "Britons"
```

Comment:
This should be replaced with Celtic, no?

Reply:
I've changed them on my end but I've yet to push to keep the PR focused.